support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution/PlaceholderTask/PlaceholderExecutable/executorCell.jelly

```diff
@@ -26,10 +26,23 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson" xmlns:l="/lib/layout">
     <td class="pane">
-        <div style="white-space: normal">${%Building}
-            <!-- TODO check for Item.READ permission on the accessControlled() -->
-            <a href="${rootURL}/${it.parent.url}" class="model-link"><l:breakable value="${it.parent.fullDisplayName}"/></a>
-            <t:buildProgressBar build="${it.parent}" executor="${executor}"/>
+        <div style="white-space: normal">
+            <j:set var="r" value="${it.parent.run()}"/>
+            <j:set var="rAuth" value="${r != null and h.hasPermission(r.parent, r.parent.READ) ? r : null}"/>
```

Comment:
<code>Item.DISCOVER</code> may be missing in the logic (e.g. "I can see the project, but cannot open its internals"). On the other hand, this executor cell will leak some info in such case